etc/osg/image-config.d/20-osg-ce-setup.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+. ce-common-startup
+
+users=$(get_mapped_users)
+for user in "$users"; do
```

Comment:
I suppose you mean

```suggestion
for user in $users; do
```

If you leave `"$users"` quoted, it will evaluate as a single string; thus a single `$user` which is the entire output of `get_mapped_users`.